queries/places.py

```diff
@@ -388,9 +389,7 @@ def sentence(state: QueryStateDict, result: Result) -> None:
         subj = result["subject_nom"]
         try:
             handlerfunc = _HANDLER_MAP[result.qkey]
-            res: Optional[AnswerTuple] = None
-            if q.location is not None:
-                res = handlerfunc(subj, q.location, result.qkey)
+            res: Optional[AnswerTuple] = handlerfunc(subj, q.location, result.qkey)
```

Comment:
Is it certain that handlerfunc() (whichever it is at runtime) is able to cope with q.location of None?

Reply:
Yes, I checked and it's covered by tests in any case. But this module needs refactoring, as noted in a new TODO at the top of the file.